src/caffe/test/test_convolution_layer.cpp

```diff
@@ -233,6 +237,7 @@ TYPED_TEST(ConvolutionLayerTest, TestSimpleConvolution) {
       layer_param.mutable_convolution_param();
   convolution_param->add_kernel_size(3);
   convolution_param->add_stride(2);
+  convolution_param->add_dilation(1);
```

Comment:
Is this necessary? I don't see a reason to be explicit about this here (and maybe there is a reason to not be explicit about it, since the common case is unspecified `dilation`...)